### PR TITLE
script(PS): add MicrosoftGraph

### PR DESCRIPTION
**Pull request by PieterHeemeryck**

### CODE REVIEW

File: PowerShell/MicrosoftGraph/MicrosoftGraph.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This script guides you through the basics of how to get started with the Microsoft Graph PowerShell SDK. This replaces the soon to be deprecated / retired AzureAD PowerShell module.
+
+## SOURCE
+https://github.com/dlw-digitalworkplace/dw-script-lib/tree/main/PowerShell/MicrosoftGraph
+
+## AUTHOR
+ - Name: Pieter Heemeryck
+ - Email: pieter.heemeryck@delaware.pro
+
+## Prerequisites
```
Comment (agtenr) [inline PowerShell/MicrosoftGraph/MicrosoftGraph.md]: The Microsoft.Graph module is required I guess?

File: PowerShell/MicrosoftGraph/MicrosoftGraph.ps1
Comment (agtenr) [inline PowerShell/MicrosoftGraph/MicrosoftGraph.ps1:1]: Nice Examples Pieter. Maybe just change the name to follow the naming conventions. (e.g. Explore-MicrosoftGraphModule ?)

**Review by agtenr — CHANGES_REQUESTED** [review]

File: PowerShell/MicrosoftGraph/MicrosoftGraph.md (hunk 1 above)
Comment (PieterHeemeryck) [reply PowerShell/MicrosoftGraph/MicrosoftGraph.md]: @agtenr Fixed

File: PowerShell/MicrosoftGraph/MicrosoftGraph.ps1
Comment (PieterHeemeryck) [reply PowerShell/MicrosoftGraph/MicrosoftGraph.ps1:1]: @agtenr Fixed

**Review by agtenr — APPROVED** [review]

File: PowerShell/MicrosoftGraph/MicrosoftGraph.ps1
Comment (agtenr) [reply PowerShell/MicrosoftGraph/MicrosoftGraph.ps1:1]: Thanks. Nice addition

**Pull request merged**